ultravox/tools/ds_tool.py

```diff
@@ -0,0 +1,172 @@
+import dataclasses
+import os
+from typing import Any, Dict, Optional, Union
+
+import datasets
+import openai
+import simple_parsing
+
+from ultravox.tools import tts
+
+CHAT_CLIENT = openai.Client()
```

Comment:
I don't think ALL_CAPS is the right naming for object like this. I think `chat_client` or `tts_client` on their own are the right choice here. Alternatively, maybe you could push these objects into their respective tasks?

Reply:
I'll see if I can do that. The `frozen=True` might cause issues for adding the object in those dataclasses.

ALL_CAPS: I thought that was the Google style, but maybe I'm wrong?
So global variables (not constant values like number/string), don't need to be ALL_CAPS?
![image](https://github.com/fixie-ai/ultravox/assets/5062458/c308a6c7-4e26-418d-9b3e-cf62469c2e04)